.babelrc

```diff
@@ -13,11 +13,5 @@
       ],
       "debug": true
     }]
-  ],
-  "plugins": [
-    // Included in the preset, but we can't specify per-plugin options there
-    ["@babel/plugin-transform-classes", {
```

Comment:
Can you drop this dependency from package.json?

Reply:
oh sure. It’s already included via the env preset but I’ll remove it anyways

Reply:
actually, there never was an explicit dep here (oops), it just worked by accident because env brings it in.

Reply:
Ooops.